### PR TITLE
design_paper and short_paper: Some edits

### PR DESCRIPTION
**Pull request by theBergmann**
Design_paper:

Rewrote the section Design Requirements -> Payment Example. 

It's a far reaching rewrite, in draft state, pulled for inspection and to keep it up to date.

Short_paper:

Added the section Design Requirements -> Delivery side.

### CODE REVIEW

File: doc/design_paper.md [hunk 1]
```diff
@@ -458,40 +458,48 @@ Any party is able to render and apply functions to the token using Tokenscript,
...
 
-DAI is a token designed for payment - purchasing security token, purchasing goods and services and so like. It's intended to match USD in value. Not fixing the supply cap, it is not itself an investment candidate.
+If we look at our property example, we see the need for some payment functions not available in current wallets: Maybe a payer needs to attach an identity file to a payment when he buys a property share. On the other side, the property token issuer might want to create recurring payouts of the income of the 1% token. If someone builds a platform to tokenize property shares, the owners will needs an easy way to integrate those kinds of payments. Further, paying for a property share might include multisig-payments, which introduce trusted third parties like notaries. A property trading platform might want to link payments to a list of certified notaries.
 
-In many ways, DAI functions like Ether, the base currency in Ethereum. However, it can't be a drop-in replacement for Ether.
```
Comment (theBergmann) [inline doc/design_paper.md]: comment

Comment (colourful-land) [reply doc/design_paper.md]: "provide identity proof", not "attach identity file"

File: doc/design_paper.md [hunk 2]
```diff
@@ -458,40 +458,48 @@ Any party is able to render and apply functions to the token using Tokenscript,
 **[[loudly thinking: Wouldn't it be better to continue with the payment side example? It somehow breaks the flow. Advantages doesn't seem to be a good fit in the chapter about requirements. Maybe I'll test it; maybe it's good to explain the advantages when we have also the payment side. The more I think about it, the more I tend to it. We already have a lot of talk about why Tokenscript. So we better stick to the examples and present the advantages later on - I tried it out.]]
```
Comment (colourful-land) [inline doc/design_paper.md]: Can you put this in a comment:
method: https://stackoverflow.com/questions/4823468/comments-in-markdown

File: doc/design_paper.md [hunk 3]
```diff
@@ -514,7 +522,9 @@ Vertically, a structured transaction is built using a token transaction or creat
 
 #### Security
 
-It is impractical to improvise a schema where every transaction the user might sign is rendered in a user-readable format. It's easy to start with such an effort with a transaction visualiser tool, interpreting an enigmatic transaction payload to the user, similar to Linux's `identify(1)` command, but ultimately the system integration and UX needs would surpass what a dictionary style transaction visualiser can do.
+It is impractical to improvise a schema where every transaction the user might sign is rendered in a user-readable format. It's easy to start with such an effort with a transaction visualiser tool, interpreting an enigmatic transaction payload to the user, similar to Linux's `identify(1)` command, but ultimately the system integratiwie sieht es mit dem Shop aus? Ich würde gern das PDF promoten und Email-Adressen sammeln.
```
[the remark this answers is not in the document]
Comment (theBergmann) [reply doc/design_paper.md]: Oh, this is a bit messy, but this part is WIP

File: doc/design_paper.md [hunk 4]
```diff
@@ -458,40 +458,48 @@ Any party is able to render and apply functions to the token using Tokenscript,
...
 
-If Pizza website doesn't upgrade, the user has to convert DAI to Ether first, then, purchase Pizza with a second transaction. Such a process is not only inconvenient, but lack atomicity, meaning that the user could have converted DAI to Ether through the effort, only to fail the checkout since the Pizza is sold out, and ended up with additional Ethers with which she has to deal.
+In the future, the payment side of smart contract blockchains like Ethereum is expected to fundamentally: There are ideas to introduce new IBAN like address schemeres or to put payments to domain names of Ethereum Name Service (ENS).We already see smart contracted payment request providers and offchain payment railways, like Raiden or Plasma.
```
[the remark this answers is not in the document]
Comment (theBergmann) [reply doc/design_paper.md]: "to fundamentally change and advance" (missed a word)

File: doc/design_paper.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (colourful-land) [reply doc/design_paper.md]: Marked as WIP. I will merge it in when it's no longer WIP. If you want to start another batch of work without finishing this, you will need to learn git branch.

**Comment by James-Sangalli** [comment]
@colourful-land please rebase!!!

**Pull request merged**